### ISSUE TITLE
Background workstation garbage collection

### ISSUE THREAD

**Issue by helloguo** [dotnet/docs#8008]
In the picture of 'Background workstation garbage collection', the activities of the separate dedicated 'GC THREAD' is shown as ALLOCATING-&gt;GC-&gt;ALLOCATING. Should it be WAITING-&gt;GC-&gt;WAITING?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d8948cf5-0aec-308c-8046-965cc15763b2
* Version Independent ID: a30a0be5-b741-2a9c-e997-55d6d9960ca9
* Content: [Fundamentals of Garbage Collection](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/fundamentals)
* Content Source: [docs/standard/garbage-collection/fundamentals.md](https://github.com/dotnet/docs/blob/master/docs/standard/garbage-collection/fundamentals.md)
* Product: **dotnet**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

**Comment by BillWagner** [comment]
@Maoni0 Can you look at this issue, and let us know if the figure should be changed.

**Comment by Maoni0** [comment]
yes! that should be changed. that looks like a typo from whoever made that picture.
